nearcore/src/migrations.rs

```diff
@@ -20,7 +20,6 @@ use near_primitives::types::{BlockHeight, ShardId};
 use near_store::db::DBCol::ColReceipts;
 use near_store::migrations::{set_store_version, BatchedStoreUpdate};
 use near_store::{create_store, DBCol, StoreUpdate};
-use std::path::Path;
```

Comment:
It's idiomatic to import `Path` and `PathBuf`.

Reply:
fixed